lib/utils/utils.js

```diff
@@ -222,6 +223,27 @@ function getAncestor(node, predicate) {
   return null;
 }
 
+/**
+ * Examples:
+ * x -> x
+ * x() -> x
+ * x.y -> x.y
+ * x.y() -> x.y
+ *
+ * @param {node} node
+ * @returns {string}
+ */
+function getName(node) {
+  if (isIdentifier(node)) {
+    return node.name;
+  } else if (isCallExpression(node)) {
```

Comment:
Should this include optional call and member expressions as well?

Reply:
Probably yes, in the future. I'm going to leave it out for now since I don't have a use case for it yet.

Reply:
I think it would be better to go ahead and add them, since you're adding it as a util instead of adding it specifically for this rule.

Reply:
Is `@computed?.readOnly('foo')` valid? It doesn't make sense, but if it's valid we should handle it.

Reply:
Added support for optional expressions. But verified that `?.` is not valid in a decorator.

Reply:
Oh good. That would have been silly :p